Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -285,6 +285,24 @@ def is_valid_type(_type):
             DBotScoreReliability.F,
         )
 
+    @staticmethod
+    def get_dbot_score_reliability_from_str(reliability_str):
```

Comment:
Also, add a docstring.
```suggestion
    def get_dbot_score_reliability_from_str(reliability_str: str):
```